iothub/device/src/Transport/Mqtt/ClientWebSocketChannel.cs

```diff
@@ -65,7 +65,7 @@ public void Dispose()
             _writeCancellationTokenSource = null;
         }
 
-        protected class WebSocketChannelUnsafe : AbstractUnsafe
+        private class WebSocketChannelUnsafe : AbstractUnsafe
```

Comment:
When building I saw a warning here that the parent class is sealed, so this type being protected made no sense.